spec/lib/hubspot/task_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe Hubspot::Task do
+  describe 'create!' do
+    subject(:new_task) do
+      params = {
+        hs_task_body: 'i am a task',
+        hs_task_subject: 'title of task',
+        hs_timestamp: DateTime.now.strftime('%Q')
+      }
+      described_class.create!(params, ticket_id: 16174569112)
```

Comment:
I wonder how much of a façade this lib should be.
I mean, it would be nicer to users if the `create!` signature was like `(body, subject, ticket_id, timestamp: DateTime.now)` and the gem would do the work of building the hash

Reply:
[seems to be a bad idea after all](https://github.com/captaincontrat/hubspot-api-ruby/pull/24/files/d9866824de917b2bf54b1b853343cfd4b9847bc9#diff-b5db0808ed7f0be8940654cde65b6d5bd57dc3e0dac4f7b921ce6c72985c371c)